migrations/sqls/20200108170348-add-undertaker-and-regions-to-licences-up.sql

```diff
@@ -0,0 +1,11 @@
+-- no live task will currently populate licences
+-- so this is required for developers who may have
+-- licence data for local testing.
+truncate water.licences cascade;
+
+alter table water.licences
+  add column is_water_undertaker boolean not null,
+  add column regions jsonb not null,
+  add column date_created timestamp not null default now(),
+  add column date_updated timestamp not null default now(),
+  add constraint c_licences_lic_ref_region unique (region_id, licence_ref);
```

Comment:
Licence numbers should be unique irrespective of the region (I have confirmed this in NALD data), so I think we just  need a unique constraint on the licence_ref column